frontend/public/components/app.jsx

```diff
@@ -120,133 +235,844 @@ class App extends React.PureComponent {
-            <Route path="/monitoring/alerts" exact component={AlertsPage} />
-            <Route path="/monitoring/alerts/:name" exact component={AlertsDetailsPage} />
-            <Route path="/monitoring/alertrules/:name" exact component={AlertRulesDetailsPage} />
-            <Route path="/monitoring/silences" exact component={SilencesPage} />
-            <Route path="/monitoring/silences/new" exact component={CreateSilence} />
-            <Route path="/monitoring/silences/:id" exact component={SilencesDetailsPage} />
-            <Route path="/monitoring/silences/:id/edit" exact component={EditSilence} />
-
-            <Route path="/k8s/cluster/:plural" exact component={ResourceListPage} />
-            <LazyRoute path="/k8s/cluster/:plural/new" exact loader={() => import('./create-yaml' /* webpackChunkName: "create-yaml" */).then(m => m.CreateYAML)} />
-            <Route path="/k8s/cluster/:plural/:name" component={ResourceDetailsPage} />
-            <LazyRoute path="/k8s/ns/:ns/pods/:podName/containers/:name" loader={() => import('./container').then(m => m.ContainersDetailsPage)} />
-            <LazyRoute path="/k8s/ns/:ns/:plural/new" exact loader={() => import('./create-yaml' /* webpackChunkName: "create-yaml" */).then(m => NamespaceFromURL(m.CreateYAML))} />
-            <Route path="/k8s/ns/:ns/:plural/:name" component={ResourceDetailsPage} />
-            <Route path="/k8s/ns/:ns/:plural" exact component={ResourceListPage} />
-
-            <Route path="/k8s/all-namespaces/:plural" exact component={ResourceListPage} />
-            <Route path="/k8s/all-namespaces/:plural/:name" component={ResourceDetailsPage} />
-
-            <LazyRoute path="/settings/profile" exact loader={() => import('./profile').then(m => m.ProfilePage)} />
-            <LazyRoute path="/settings/ldap" exact loader={() => import('./cluster-settings/ldap').then(m => m.LDAPPage)} />
-            <LazyRoute path="/settings/cluster" exact loader={() => import('./cluster-settings/cluster-settings').then(m => m.ClusterSettingsPage)} />
-
-            <LazyRoute path="/error" exact loader={() => import('./error').then(m => m.ErrorPage)} />
-            <Route path="/" exact component={DefaultPage} />
-
-            <LazyRoute loader={() => import('./error').then(m => m.ErrorPage404)} />
-          </Switch>
-        </div>
-      </div>
-    </React.Fragment>;
+  render() {
+    const {
+      isDropdownOpen,
+      isKebabDropdownOpen,
+      activeItem,
+      isNavOpen,
+      activeGroup
+    } = this.state;
+
+    const PageNav = (
+      <Nav onSelect={this.onNavSelect} aria-label="Nav">
+        <NavList>
+          <NavExpandable
+            title="Home"
+            groupId="grp-1"
+            isActive={activeGroup === 'grp-1'}
+            isExpanded
+          >
+            <NavItem
+              groupId="grp-1"
+              itemId="grp-1_itm-1"
+              to="/overview/ns/default"
```

Comment:
dummy routes to enable demonstrating a few existing pages, will need to integrate these components into masthead.jsx and nav.jsx respectively...